synadm/cli/user.py

```diff
@@ -394,14 +394,14 @@ def name_extra(self):
     help="Set display name. defaults to the value of user_id")
 @optgroup.option(
     "--threepid", "-t", type=str, multiple=True, nargs=2,
-    help="""Add a third-party identifier. This can be an email address or a
-    phone number. Threepids are used for several things: For use when
-    logging in, as an alternative to the user id. In the case of email, as
-    an alternative contact to help with account recovery, as well as
-    to receive notifications of missed messages. Format: medium
-    value (eg. --threepid email <user@example.org>). This option can also
-    be stated multiple times, i.e. a user can have multiple threepids
-    configured.""")
+    help="""Add a third-party identifier (email address or phone number).
```

Comment:
So does it add 3pids or replace them in the end?

Reply:
Ah yeah, thanks for that catch! It relaces for existing accounts. That was the reason I was rewriting this help text in the first place. The problem with wording right now is that this user modify command is used to create as well as alter user accounts. Suggestions how to write this concisely?

Set